CHANGELOG.md

```diff
@@ -132,6 +132,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Add unit tests and a component spec test for the above.
   - **CUMULUS-2446**
     - Remove schema validation check against DynamoDB table for collections when migrating records from DynamoDB to core PostgreSQL database.
+  - **CUMULUS-2447**
+    - Changed `translateApiAsyncOperationToPostgresAsyncOperation` to call `JSON.stringify` and then `JSON.parse` on output.
```

Comment:
This entry (and others like it), are odd, as it's a change to a feature branch and not a true update to main.    No objection given we're doing it elsewhere, but highlighting it for upcoming merge discussion.